src/oc/web/ws/utils.cljs

```diff
@@ -6,9 +6,39 @@
             [oc.web.lib.raven :as sentry]
             [oc.web.local-settings :as ls]))
 
-;; Connection check
+;; cmd queue
+(defonce cmd-queue (atom []))
+
+(defn buffer-cmd [args]
+  (swap! cmd-queue conj args))
+
+(defn reset-queue []
+  (reset! cmd-queue []))
+
+(defn send-queue [chsk-send! ch-state]
+  (when (and chsk-send! (:open? @@ch-state))
+    (doseq [qargs @cmd-queue]
+      (apply chsk-send! qargs))
+    (reset-queue)))
+
+(declare sentry-report)
+(defn not-connected [service-name chsk-send! ch-state & args]
+  (let [arg (first (first args))]
+    (buffer-cmd arg)
+    (sentry-report service-name chsk-send! ch-state arg)))
```

Comment:
@thepug i think we can remove this, WYT?
Before it was useful to know how many messages were sent over a closed socket, because they were lost messages. Now we queue them up, so if you see another place were we could be losing messages (for example if when you go to drain the queue the socket is closed again and you lose them) then we should add a sentry report there.
Let's discuss in #development